src/inflators/link.ts

```diff
@@ -2,12 +2,23 @@ import { addComponent } from "bitecs";
 import { Link } from "../bit-components";
 import { HubsWorld } from "../app";
 
+export enum LinkType {
+  LINK = 0,
+  AVATAR = 1,
+  SCENE = 2,
+  ROOM = 3,
+  ROOM_URL = 4,
+  WAYPOINT = 5
+}
+
 export type LinkParams = {
   href: string;
+  type?: LinkType;
 };
 
 export function inflateLink(world: HubsWorld, eid: number, params: LinkParams): number {
   addComponent(world, Link, eid);
   Link.url[eid] = APP.getSid(params.href);
+  Link.type[eid] = params.type || LinkType.LINK;
```

Comment:
Nit: I prefer `params.type !== params.type ? params.type : LinkType.Link` or something similar because it would be clearer and safer.

`params.type || LinkType.LINK` works because `LinkType.LINK` is zero. But imagine a case that a future new developer changes `LinkType` enum values for a certain purpose and the one (and reviewers) don't notice there is a line relying on `LinkType.LINK` is zero.